### PR TITLE
Translated untranslated English strings into german

### PR DESCRIPTION
**Pull request by Cookie303728**
As the title says, all strings that where previously untranslated thanks to the complicated locale system are now fully translated into german.
@Kakiking you probably want to check my translations since im not 100% perfect

### CODE REVIEW

**Review by AnthonyFuller — APPROVED** [review]
Awaiting once over from @Kakiking or a second-look by @Cookie303728.

File: resources/locale.json [hunk 1]
```diff
@@ -745,195 +745,195 @@
...
         "UI_PEACOCK_PIRATESPROBLEM2": "Das Piraten Problem - Stufe 2",
         "UI_PEACOCK_PIRATESPROBLEM3": "Das Piraten Problem - Stufe 3",
-        "UI_PEACOCK_PIRATESPROBLEM_CALM": "Calm the staff",
-        "UI_PEACOCK_PIRATESPROBLEM_CALM_DESC": "Pirates have stolen code from the Haven™ servers, and the staff is in panic.\nCalm the staff by pumping a sedative in their ventilation.",
-        "UI_PEACOCK_PIRATESPROBLEM_ALARM": "Trigger an alarm",
-        "UI_PEACOCK_PIRATESPROBLEM_ALARM_DESC": "Make sure the issue doesn't escalate.",
-        "UI_PEACOCK_PIRATESPROBLEM_PIRATES": "Obtain the list of pirates",
-        "UI_PEACOCK_PIRATESPROBLEM_PIRATES_DESC": "Get the list containing all pirate identities from The Captain.",
+        "UI_PEACOCK_PIRATESPROBLEM_CALM": "Beruhige das Personal",
+        "UI_PEACOCK_PIRATESPROBLEM_CALM_DESC": "Piraten haben Code aus den Haven™ Servern gestohlen und das Personal ist in Aufruhr.\nBeruhige sie indem du Beruhigungsmittel in ihre Lüftung pumpst.",
```
Comment (Kakiking) [inline resources/locale.json:784]: I think "Code von den Servern gestohlen" sounds better. "Code aus den Servern gestohlen" sounds like they've taken it out physically.

File: resources/locale.json [hunk 2]
```diff
@@ -745,195 +745,195 @@
...
-        "UI_PEACOCK_ROSEBUSH_DISGUISE2": "We can't just sit around and do nothing Vincent! I'm going in!",
-        "UI_PEACOCK_ROSEBUSH_FINISH": "Finish it",
-        "UI_PEACOCK_ROSEBUSH_FINISH_DESC": "It's all out war now.",
+        "UI_PEACOCK_ROSEBUSH_MANSION_GUARDS": "Die Anwesenswachen",
+        "UI_PEACOCK_ROSEBUSH_MANSION_GUARDS_DESC": "Diese Möchtegerncops dachten tatsächlich sie könnten einfach ins Carlisle Anwesen kommen. Du hättest ihre Gesichter sehen sollen als wir ihnen gesagt haben das die draußen bleiben sollen!",
+        "UI_PEACOCK_ROSEBUSH_MANSION_GUARDS_HUD": "Eliminiere die 2 Anwesenswachen",
+        "UI_PEACOCK_ROSEBUSH_MANSION_STAFF": "Die Anwesenswachen",
+        "UI_PEACOCK_ROSEBUSH_MANSION_STAFF_DESC": "Wir haben gerade mitbekommen das wir zwei unserer Leute verloren haben. Ich weiß das diese arroganten Arschlöcher dahinter stecken!",
+        "UI_PEACOCK_ROSEBUSH_MANSION_STAFF_HUD": "Eliminiere 2 Anwesenswachen",
+        "UI_PEACOCK_ROSEBUSH_DISGUISE1": "Zeige diesen Niederrangigen Rüpeln eine oder zwei Lektionen über Klasse und lass es sie wissen das es von uns kommt.",
```
Comment (Kakiking) [inline resources/locale.json:799]: It should be "niederrangigen" here.

File: resources/locale.json [hunk 3]
```diff
@@ -745,195 +745,195 @@
...
-        "UI_PEACOCK_HEDGEBUSH_GOLD_DESC": "Steal all the gold from the vault.",
-        "UI_PEACOCK_HEDGEBUSH_BURGERS": "Steal the Burgers",
-        "UI_PEACOCK_HEDGEBUSH_BURGERS_DESC": "Steal all the burgers from the bank.",
-        "UI_PEACOCK_HEDGEBUSH_OBJ1": "Eliminate Frank Winkelman with a Cheeseburger as the Bank Robber",
-        "UI_PEACOCK_HEDGEBUSH_OBJ2": "Eliminate Gourd with a Cheeseburger",
+        "UI_PEACOCK_HEDGEBUSH_GOLD": "Stehle das Gold",
+        "UI_PEACOCK_HEDGEBUSH_GOLD_DESC": "Stehle das komplette Gold aus dem Tresor.",
+        "UI_PEACOCK_HEDGEBUSH_BURGERS": "Stehle die Burger",
+        "UI_PEACOCK_HEDGEBUSH_BURGERS_DESC": "Stehle alle Burger aus der Bank.",
+        "UI_PEACOCK_HEDGEBUSH_OBJ1": "Eliminiere Frank Winkelman mit einem Cheeseburger als Bank Räuber verkleidet",
```
Comment (Kakiking) [inline resources/locale.json:822]: That should be "Bankräuber".

File: resources/locale.json [hunk 4]
```diff
@@ -745,195 +745,195 @@
...
         "UI_PEACOCK_GRAPEBUSH2": "Die Argentine Bitterkeit - Stufe 2",
         "UI_PEACOCK_GRAPEBUSH3": "Die Argentine Bitterkeit - Stufe 3",
-        "UI_PEACOCK_GRAPEBUSH_FILES": "Get the Leak Report from the Basement Safe",
-        "UI_PEACOCK_GRAPEBUSH_FILES_DESC": "Get the Leak Report from the Basement Safe.",
-        "UI_PEACOCK_GRAPEBUSH_OBJ1": "Eliminate Violetta Mendez in an accident",
-        "UI_PEACOCK_GRAPEBUSH_OBJ2": "Eliminate Montel Beneventi in an explosion or with a cannon",
-        "UI_PEACOCK_GRAPEBUSH_OBJ2_DESC": "Eliminate Montel Beneventi in an explosion or with a cannon.",
-        "UI_PEACOCK_GRAPEBUSH_OBJ3": "[Optional] Eliminate Hira Amala",
-        "UI_PEACOCK_GRAPEBUSH_OBJ3_DESC": "If you're up for it, eliminate Hira Amala.",
+        "UI_PEACOCK_GRAPEBUSH_FILES": "Besorge den Leckbericht aus dem Kellertresor",
```
Comment (Kakiking) [inline resources/locale.json:839]: "Leckbericht" sounds weird, the first thing I think of when I hear that is licking lmao

File: resources/locale.json [hunk 5]
```diff
@@ -745,195 +745,195 @@
...
         "UI_PEACOCK_SOUTHERN_COMFORT": "Das Weihnachts Unglück",
         "UI_PEACOCK_SOUTHERN_COMFORT1": "Das Weihnachts Unglück - Stufe 1",
         "UI_PEACOCK_SOUTHERN_COMFORT2": "Das Weihnachts Unglück - Stufe 2",
         "UI_PEACOCK_SOUTHERN_COMFORT3": "Das Weihnachts Unglück - Stufe 3",
-        "UI_PEACOCK_SOUTHERN_COMFORT_OBJ1": "Eliminate at least 5 of the possible targets outlined in-game by pacifying them or killing them with a Christmas-themed weapon.",
-        "UI_PEACOCK_SOUTHERN_COMFORT_SANTA_TITLE": "Become Santa",
-        "UI_PEACOCK_SOUTHERN_COMFORT_SANTA_DESC": "Disguise yourself as Santa 47, do not change disguises after.",
-        "UI_PEACOCK_SOUTHERN_COMFORT_SANTA_HUD": "Become Santa, stay as Santa",
-        "UI_PEACOCK_SOUTHERN_COMFORT_KILL_SANTA": "Eliminate Santa",
+        "UI_PEACOCK_SOUTHERN_COMFORT_OBJ1": "Eliminiere mindestens 5 markierte Ziele indem du sie mit einer Weihnachtsthema Waffe tötest oder ruhigstellst.",
```
Comment (Kakiking) [inline resources/locale.json:853]: "Weihnachtsthema Waffe" doesn't make any sense in my opinion. "Weihnachtswaffe" would probably be better.

File: resources/locale.json [hunk 6]
```diff
@@ -745,195 +745,195 @@
...
+        "UI_PEACOCK_SOUTHERN_COMFORT_SANTA_TITLE": "Werde Santa",
+        "UI_PEACOCK_SOUTHERN_COMFORT_SANTA_DESC": "Verkleide dich als Santa 47, wechsle danach deine Verkleidug nichtmehr.",
```
Comment (Kakiking) [inline resources/locale.json:855]: "nicht mehr", those should be seperate words.

File: resources/locale.json [hunk 7]
```diff
@@ -745,195 +745,195 @@
...
-        "UI_PEACOCK_LEEHONG_RIFLE": "The Gold Dragon",
-        "UI_PEACOCK_LEEHONG_RIFLE_DESC": "Retrieve The Gold Dragon Rifle.",
-        "UI_PEACOCK_LEEHONG_RIFLE_HUD": "Retrieve The Gold Dragon Rifle",
-        "UI_PEACOCK_LEEHONG_ANGUS": "Eliminate Angus Pritchard with The Gold Dragon Rifle",
-        "UI_PEACOCK_LEEHONG_DRIVER": "The Driver",
-        "UI_PEACOCK_LEEHONG_DRIVER_DESC": "Eliminate Lee Hong's driver Blue Lotus in an explosion.",
-        "UI_PEACOCK_LEEHONG_DRIVER_HUD": "Eliminate the Blue Lotus in an explosion",
-        "UI_PEACOCK_POWERED_BY": "Powered by",
+        "UI_PEACOCK_LONGBUSH_GYM_ACCIDENT": "Fitnessstudio Unfall",
+        "UI_PEACOCK_LONGBUSH_GYM_ACCIDENT1_DESC": "Eliminiere Arrush Lal als Personal Trainer verkleidet nachdem du ihn mit einem Trainingsgewicht ruhiggestellt hast",
```
Comment (Kakiking) [inline resources/locale.json:863]: "Personal Trainer" sounds like a trainer for staff. "persönlicher Trainer" would be better.

File: resources/locale.json [hunk 8]
```diff
@@ -745,195 +745,195 @@
...
+        "UI_PEACOCK_TRADITIONS_BOMB": "Entschärfe die Bombe",
+        "UI_PEACOCK_TRADITIONS_BOMB_DESC": "Entschärfe die Bombe die im Hotel versteckt ist.",
+        "UI_PEACOCK_TRADITIONS_CONTACT": "Der Kontakt",
+        "UI_PEACOCK_TRADITIONS_CONTACT_DESC": "Stelle den Kontakt weder ruhig noch töte ihn.",
+        "UI_PEACOCK_TRADITIONS_CONTACT_HUD": "Stelle den Kontakt weder ruhig noch töte ihn.",
+        "UI_PEACOCK_TRADITIONS_KEYCARD": "Raum Schlüsselkarte",
+        "UI_PEACOCK_TRADITIONS_KEYCARD_DESC": "Gehe zur Rezeption und checke als Tobias Reaper ein. Warte dann auf weitere Anweisungen wie du den Kontakt triffst.",
+        "UI_PEACOCK_TRADITIONS_KEYCARD_HUD": "Besorge deine Zimmerkarte",
+        "UI_PEACOCK_TRADITIONS_207KEYCARD": "Raum 207",
+        "UI_PEACOCK_TRADITIONS_207KEYCARD_DESC": "Gehe in den ersten Stock der Bar, tauche als Kunde unter und triff den Kontakt um die Zimmerkarte des Ziels zu erhalten. Sie nennt dich ihren \"Exehemann\".",
```
Comment (Kakiking) [inline resources/locale.json:888]: It should be "Ex-Ehemann", shouldn't it?

File: resources/locale.json [hunk 9]
```diff
@@ -745,195 +745,195 @@
...
+        "UI_PEACOCK_TRADITIONS_207KEYCARD_HUD": "Besorge die Karte zu Zimmer 207",
+        "UI_PEACOCK_TRADITIONS_JULIAN": "Eliminiere Julian",
+        "UI_PEACOCK_TRADITIONS_JULIAN_HUD": "Eliminiere Julian mit einem Elektroschockunfall",
+        "UI_PEACOCK_TRADITIONS_JULIAN_DESC": "Jacke Carrington hat das Ziel in seinem Raum betäubt. Eliminiere ihn. Du musst es wie einen Elektrounfall aussehen lassen sonst wird verdacht erregt.",
+        "UI_PEACOCK_TRADITIONS_STALKER": "Der Stalker",
+        "UI_PEACOCK_TRADITIONS_STALKER_DESC": "47, es scheint als ob jemand deinen Kontakt beobachtet. Er saß neben dir als du dich mit ihr getroffen hast. Stelle ihn ruhig und stecke ihn in einem Behälter zur Extraktion und Befragung.",
+        "UI_PEACOCK_TRADITIONS_STALKER_HUD": "Stelle den Stalker ruhig und verstecke ihn",
+        "UI_PEACOCK_TRADITIONS_MILITIA": "Die Miliz",
+        "UI_PEACOCK_TRADITIONS_MILITIA1_DESC": "Er ist nicht alleine... Eliminiere alle Miliz Soldaten in der Medow Brown Lounge um den Bereich zu säubern.",
```
Comment (Kakiking) [inline resources/locale.json:897]: It should either be "Miliz-Soldaten" or "Milizsoldaten".

File: resources/locale.json [hunk 10]
```diff
@@ -745,195 +745,195 @@
...
+        "UI_PEACOCK_TRADITIONS_ACCOMPLICE_HUD": "Eliminiere den Komplizen",
+        "UI_PEACOCK_LEEHONG": "Das Lee Hong Attentat",
+        "UI_PEACOCK_LEEHONG_DESC": "Guten Abend 47. Unser Ziel ist Lee Hong, der Anführer der Red Dragon Triade und Angus Pritchard, ein reicher Antiquitätensammler aber auch eine ehemalige FBI Quelle.Beide sollen sich in ein paar Minuten in einem Fisch Restraurant Treffen um ein goldenes Idol für Informationen über unseren Klienten zu tauschen.Der Klient möchte das du diese Informationen besorgst.Hong und Pritchard möchten nicht zusammen gesehen werden also essen sie an getrennten Tischen, aber sie werden den Austausch in ungefähr 10 Minuten durchführen.Das darf nicht passieren, 47.Unser Klient möchte das sie im Glauben gelassen werden, das der Tausch zwischen den Beiden schiefging um keine Aufmerksamkeit zu erregen.Sei kreativ.Ich habe gehört die Triade benutzt spezielle Munition für ihre Gewehre.Vielleicht kannst du diese Informationen nutzen um die Triade mit Pritchards Mord zu beschuldigen...\nDie Uhr tickt 47. Viel Glück.\nGemacht von einem Peacock Community Mitglied bekannt als kercyx.",
```
Comment (Kakiking) [inline resources/locale.json:912]: Also it's "Fischrestaurant". Also you didn't add a space after the end of a sentence multiple times here.

**Review by Kakiking — CHANGES_REQUESTED** [review]

**Review by Kakiking — APPROVED** [review]
Thanks for the contribution :+1:

**Pull request merged**